app/models/league.rb

```diff
@@ -0,0 +1,5 @@
+class League < Event
+    enum game_modes: [:round_robin, :two_halfs, :swiss, :danish]
```

Comment:
What is the purpose of separate League and Tournament *classes* when the only difference is which game modes are allowed?

Reply:
From our POs explanations, we concluded that League and Tournament will have different behavior later on. Currently, you could even see the acceptance of different game modes as different behavior. Thus, I decided to create to separate classes (rather than managing the acceptance inside one class).

Is there any other reason, why we should keep those classes together?